docs/combine.rst

```diff
@@ -39,6 +41,8 @@ Commands
     Search all stockpiles.
 ``here``
     Search the currently selected stockpile.
+``item``
+    Search the currently selected item.
```

Comment:
by "item" do you mean "container"?

Reply:
any item that could contain anything, yeah